cmake/scripts/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 # This file is part of the dune-stuff project:
 #   https://github.com/wwu-numerik/dune-stuff
-# Copyright holders: Rene Milk, Felix Schindler
+# The copyright lies with the authors of this file (see below).
 # License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+# Authors:
+#   Felix Schindler (2014)
+#   Rene Milk       (2012 - 2013)
+
```

Comment:
Well, I _thought_ it would be referenced (ie add_subdir) in /cmake/CMakeLists.txt
BUt that file doesn't even exist. If you want to delete cmake/scripts/CMakeLists.txt you need to modify /CMakeLists.txt accordingly.